website/docs/usage.md

```diff
@@ -0,0 +1,42 @@
+Specifies a reusable block of configuration to be consumed at a later point.
+
+```lua
+usage 'MyUsage'
+    defines { 'MY_DEFINE' }
+```
+
+The `usage` API is used to define configuration to be consumed by the `uses` API.  Usages must have unique names, except for magic usage block names (as described below).
+
+### Magic Usage Blocks ###
+
+1. `PRIVATE` - Private specifies a configuration to be automatically applied to the project defining the `usage` block. This block is inaccessible to be consumed by `uses`.
```

Comment:
Would it make sense to say it's equivalent to adding it outside of the usage block?